contrib/checkstyle/test/resources/compatible-scala/src/com/etsy/sbt/TestClass.scala

```diff
@@ -0,0 +1 @@
+// no content since file should be ignored
```

Comment:
To make this test case a bit more robust, how about putting some garbage syntax/compile/etc. errors in this file? Right now since the file is empty, whether it is ignored or not by checkstyle doesn't affect the test. If the file compiles errors and checkstyle still passes, that tells us that the logic ignoring `.scala` files is working correctly

Reply:
The logic in Checkstyle already filters non Java files.

I added proper content though. The module name does say compatible.

Reply:
I know the logic already filters non-java files, I mean to update this `TestClass.scala` to help assert that it filters non-java files, such that if the filtering logic is buggy the test will fail due to finding non-valid-java-code in `TestClass.scala`